### PR TITLE
Change repeating elements to zero

### PR DESCRIPTION
**Pull request by YESENIA2012**
Algorithm for zeroing the elements that are equal to the element in the first position

### CODE REVIEW

File: changeVectorElementZero.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+let lea = require("prompt-sync")();
+var vector = [1, 2, 1, 3, 4, 1, 5, 6, 1];
+var contador = 1;
+var primerElemento = vector[0];
+console.log(primerElemento);
```
Comment (erick2014) [inline changeVectorElementZero.js]: please remove this console.log

**Comment by YESENIA2012** [comment]
Can you review it please

**Review by erick2014 — CHANGES_REQUESTED** [review]

File: changeVectorElementZero.js (hunk 1 above)
Comment (YESENIA2012) [reply changeVectorElementZero.js]: console.log has already been removed

**Comment by erick2014** [comment]
well done!

**Pull request merged**